src/test/java/com/geronimo/MessageServiceTest.java

```diff
@@ -46,6 +47,31 @@ public void after() {
 
     @Test
     @Transactional
+    public void testListFeedMessagesServiceMethod() {
+        author.addFollower(author); //I'm following myself, why not?
```

Comment:
This should be prohibited, liking yourself is ok, but following, oh no, this is too dangerous ;(

Reply:
working on it